### DISCUSSION TITLE
I need to run only NIDS and zeek containers - how do I achieve it

### DISCUSSION THREAD

**Discussion by Blason** [Security-Onion-Solutions/securityonion#11311] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

32

### Storage for /

500

### Storage for /nsm

800

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Hi Team,

I need a help on 2.4 image. How do I stop and disable all the components except suricata/zeek/stenographer - and send those logs to other 3rd party Elasticsearch instance. I purely wanted to collect a network logs and containers pertaining to it and not with host logs. So that my hardware utilization then will be very less and can capture the packet perfectly.

Can someone please guide me?

TIA
Blason R

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment, marked as answer]
You can disable some components via the config interface but it won't get you what you are looking for. Best bet would be to have logstash on the manager to send the data to the other elastic instance. Keep in mind that if you do this most of the functionality of SOC will not work correctly.
